roamance/src/components/social/comment/comment-form.tsx

```
@@ -0,0 +1,307 @@
+                        </motion.div>
+                      )}
+
+                      {videoPath && (
+                        <motion.div
+                          whileHover={{ scale: 1.05, y: -2 }}
+                          className="relative w-20 h-16 rounded-lg overflow-hidden border border-gray-200/80 dark:border-gray-700/60 group shadow-sm"
+                        >
+                          <video
+                            src={videoPath}
+                            className="w-full h-full object-cover"
+                          />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)